#Задача 1#

Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

▁x±z_(α/2) * σ/√n 

Z = 1,96

In [ ]:
import numpy as np
import scipy.stats as stats
M = 80
n = 256
sigma = 16
Z = stats.norm.ppf(0.975)
print(M - Z * sigma/np.sqrt(n))
print(M + Z * sigma/np.sqrt(n))

78.04003601545995
81.95996398454005


**Доверительный интервал (78.04, 81.96)**

#Задача 2#

В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные:
6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1

Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

t для 0,975 и 9 степеней свободы = 2,262

In [ ]:
import numpy as np
import scipy.stats as stats
X = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
average = np.mean(X)
D = np.var(X,ddof=1)
t = stats.t.ppf (0.975,9)
n = 10
print(average - t * np.sqrt(D/n))
print(average + t * np.sqrt(D/n))

6.267515851415713
6.912484148584288


**Истинное значение величины X лежит в доверительном интервале [6.27, 6.91] с доверительной вероятностью 0,95**

#Задача 3#

Рост дочерей 175, 167, 154, 174, 178, 148, 160, 167, 169, 170

Рост матерей  178, 165, 165, 173, 168, 155, 160, 164, 178, 175

Используя эти данные построить 95% доверительный интервал для разности среднего роста родителей и детей.

D=1/2(D_1+D_2), где   D_1 и D_2 – дисперсии обеих групп

S_Δ=√(D/n_1 +D/n_2 ), где n_1 и  n_2 – объемы выборок.

Δ± t_(α/2)* S_Δ , где Δ= ▁(X_1 )- ▁(X_2 )

In [ ]:
import numpy as np
import scipy.stats as stats
mothers = np.array([175, 167, 154, 174, 178, 148, 160, 167, 169, 170]) 
daughters = np.array([178, 165, 165, 173, 168, 155, 160, 164, 178, 175])
mothers_av = np.mean(mothers)
daughters_av = np.mean(daughters)
D1 = np.var(mothers,ddof=1)
D2 = np.var(daughters,ddof=1)
D = 1/2*(D1+D2)
S_d = np.sqrt(D/len(mothers)+D/len(daughters))
delta = mothers_av - daughters_av
t= stats.t.ppf(0.975,18)
print(delta + t*S_d)
print(delta - t*S_d)

6.268418034506846
-10.068418034506857


**95% доверительный интервал для разности среднего роста родителей и детей (-10.07, 6.27)**